### PR TITLE
bug #134170 improved navigation with keys when having multiple dropdo…

### PR DESCRIPTION
**Pull request by Cmoen11**
…wns inside of a dropdown

### CODE REVIEW

File: src/components/Controls/Dropdown/Dropdown.js [hunk 1]
```diff
@@ -63,22 +63,31 @@ class Dropdown extends React.Component {
 	handleKeyDown = e => {
 		if (
 			keyCodes.tab === e.which ||
-			(/button/i.test(e.target.tagName) && e.which === keyCodes.space) ||
-			/input|textarea/i.test(e.target.tagName)
+			(/button/i.test(e.target.tagName) && [keyCodes.space, keyCodes.enter].includes(e.which)) ||
+			/input|textarea|select/i.test(e.target.tagName)
```
Comment (Kashkovsky) [inline src/components/Controls/Dropdown/Dropdown.js]: Mouse click on <DropdownItem tag="select"> will close dropdown menu, so user won't be able to select anything.

Comment (Cmoen11) [reply src/components/Controls/Dropdown/Dropdown.js]: Ok, i see. Possible solution is <DropdownItem tag="select" toggle={false}> on the menuItem. Do you have another solution?

Comment (Kashkovsky) [reply src/components/Controls/Dropdown/Dropdown.js]: Will it work within a keyboard? Anyway I think it's not good to expose internal logic through props. It should be handled inside.

Comment (Cmoen11) [reply src/components/Controls/Dropdown/Dropdown.js]: Yes, it'll still work. Though, I can see if I can handle this aswell.

Comment (Cmoen11) [reply src/components/Controls/Dropdown/Dropdown.js]: pushed a suggestion change in DropdownItem.js

Comment (Kashkovsky) [reply src/components/Controls/Dropdown/Dropdown.js]: That's the right fix) One more thing: click/Enter on item in nested menu should close the menu (as it works on main menu).

Comment (Cmoen11) [reply src/components/Controls/Dropdown/Dropdown.js]: I've added a prop on dropdown menu that allows parentToggle to be passed.

**Pull request merged**